app/web/pages/_document.tsx

```diff
@@ -20,6 +20,12 @@ export default class MyDocument extends Document {
             rel="stylesheet"
             href="https://cdn.couchers.org/fonts/wordmark/wordmark.css"
           />
+          {process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY && (
+            <script
+              async
+              src={`https://www.google.com/recaptcha/enterprise.js?render=${process.env.NEXT_PUBLIC_RECAPTCHA_SITE_KEY}`}
+            ></script>
+          )}
```

Comment:
So if this is meant to only run it on prod, I think better to use `NEXT_PUBLIC_COUCHERS_ENV`